types/wlr_surface.c

```diff
@@ -386,6 +386,7 @@ struct wlr_surface *wlr_surface_create(struct wl_resource *res,
 		wl_resource_post_no_memory(res);
 		return NULL;
 	}
+	wlr_log(L_DEBUG, "New wlr_surface %p (res %p)", surface, res);
```

Comment:
Not sure about these added logs, as said in the commit message it's probably not something we want all the time, but it helped me debug a bit.